### DISCUSSION TITLE
Feature request: Admins could see all collections even private ones not created by you

### DISCUSSION THREAD

**Discussion by matbrgz** [outline/outline#1428] [General] [answered]
A platform administrator can view any document in any collection.

**Comment by tommoor** [comment]
"Super Admin"

**Comment by matbrgz** [comment]
At this moment I think to keep simple, then add a role based access control.

![image](https://user-images.githubusercontent.com/23284276/89721715-582fed80-d9b7-11ea-939c-eb733cbbf3d7.png)

**Comment by guilherme-diniz** [comment]
So, for now it's just give the ability to see all to platform admins, or create a new role (superAdmin) ?

**Comment by tommoor** [comment, marked as answer]
I wrote some thoughts up on implementation here:

https://inside.getoutline.com/share/bca8a30d-be96-4fdd-a838-f6453cdd0744

**Reply by Zilborg** [reply]
Hello!
Firstly, thank you for your beautiful tool for collaboration!)

Does any update about "Super Admin" role for monitoring all created collection?
I'm worried about case, that admin doesn't see Member collection with `no access` option.
In my opinion, it can be led to uncontrolled DB and file storage overloading.
Currently, I'm thinking about the script, that will add admins to any new collection or monitoring DB tables of new collection. However, they both aren't fine solutions...
Can you comment any solution or idea for this case?

**Reply by Zilborg** [reply]
Now I found more lovely solution. Add Admin to policies for read.  ([link to code](https://github.com/outline/outline/blob/main/server/policies/collection.js#L28))
``` javascript
if (user.isAdmin) return true;
``` 

Also it's interesting, that Admin can't see all Collections, but can delete them. Is it bug? 😄  ([link to code](https://github.com/outline/outline/blob/main/server/policies/collection.js#L116))

**Reply by Zilborg** [reply]
Of course, it didn't work properly, so I added a script for adding Admin with hook of the Collection.
My [hash](https://github.com/Zilborg/outline/commit/7ba6fc74cf1dfe3870d551dce0a04a2e818cc72f).

**Reply by Zilborg** [reply]
I hope my messages don't annoy you. I think, this is the last in this topic.
The last modification. I added policy for restricted remove or change permissions of Admin accounts.
My [hash commit](https://github.com/Zilborg/outline/commit/b870f74a68243c8b6ec92427ed3d60a79b82100a).

**Reply by tommoor** [reply]
There hasn't been an update in the product as you've seen, the usecase hasn't really been one that's come up – although I'm sure it will become important eventually.

Another option would be to have a team level option to make it so that collection creation is only available to admins.

**Comment by sncat** [comment]
It's been a while since this discussion, but do you have any plans for the implementation of [owner]?

We have multiple Admin users managing the outline,
When other Admin users create [inaccessible collections], by default I cannot see that they have collections.

It would be easier to manage the entire collection if you could add an [owner] to manage all collections by default.

**Reply by tommoor** [reply]
Hi – this feature was actually built, it is one of the advanced management controls available in the [licensed edition](https://www.getoutline.com/pricing). There are no plan to bring it to the community edition.

**Reply by sncat** [reply]
Thank you for your reply!
I will consider the licensed version.
